slatedb/src/garbage_collector/wal_gc.rs

```diff
@@ -109,16 +108,12 @@ impl GcTask for WalGcTask {
         Ok(())
     }
 
-    fn interval(&self) -> Duration {
+    fn period(&self) -> Duration {
```

Comment:
should we keep the name? underlying option is still called interval

Reply:
Yea, I was torn on this. The _actual_ name in tokio is called `period`, but we misnamed it in our configs and code as `interval`. :/ I suppose to keep our codebase coherent we should keep it as-is. I'll rename back to `interval`.